presbydora.module

```diff
@@ -273,6 +285,21 @@ function presbydora_embargo_date_set_year_range($form_element, $form_values) {
  */
 function presbydora_islandora_datastream_access($op, $islandora_datastream, $user) {
   if ($islandora_datastream->id == 'OBJ' || $islandora_datastream->id == 'PDF') {
+    // Modified to allow specific objects to be accessible.
+    $object = $islandora_datastream->parent;
+    $pid = $object->id;
+
+    module_load_include('inc', 'presbydora', 'includes/utilities');
+    $bypass_pids = variable_get('presbydora_access_bypass_pids', array());
+    if (!empty($bypass_pids)) {
+      $ancestors = presbydora_get_ancestors($pid);
+      $is_intersect = !empty(array_intersect($ancestors, $bypass_pids));
+
+      if ($is_intersect) {
+        $access = (islandora_namespace_accessible($pid) && user_access($op, $user));
```

Comment:
This... isn't really necessary... Logic equivalent to that here will already be executed via [the existing access check process as linked](https://github.com/discoverygarden/presbydora/pull/14#discussion_r235753939).

It should suffice to `return NULL` here...

... the access hooks implement three-valued logic... `TRUE` indicates we want to explicitly allow things, `FALSE` to explicitly deny, `NULL` to make no assertion... All results are gathered into an array... This array must contain at least one `TRUE`  no `FALSE` in order to pass... `NULL` makes no assertion, but would allow us to avoid our explicit denial for the guest role below, allowing the request if it would otherwise be allowed.

Reply:
Thanks for the clarification